<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ObjectWidget</class>
 <widget class="QWidget" name="ObjectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>419</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insert object</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="1" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Frames</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>From:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="fromBox"/>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>To:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QSpinBox" name="toBox"/>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="curFromButton">
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="curToButton">
        <property name="text">
         <string>Current</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLineEdit" name="imgEdit"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="imgButton">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QPushButton" name="negButton">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/res/cancel.png</normaloff>:/res/cancel.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="posButton">
     <property name="text">
      <string>Apply</string>
     </property>
     <property name="icon">
      <iconset resource="../res.qrc">
       <normaloff>:/res/ok.png</normaloff>:/res/ok.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
